science/fldigi/Portfile

```diff
@@ -21,6 +21,7 @@ master_sites        http://www.w1hkj.com/files/${name}/
 checksums           rmd160  3eade162052641401bfbf5628816a897d96257b9 \
                     sha256  ed5320619594911e8b7a887a8a098ee2b08bd25a594cd0e7be8c5834ea99d224 \
                     size    4681621
+revision            1
```

Comment:
normally we place the revision line just below the version, as the two are related. Placing down here it will be easy to miss when updating the version in the future, so please move this up.

Reply:
every version change you also need to change checksums so you see that; I generally use that position with michaelld. anyway, I fixed.